vllm/engine/async_llm_engine.py

```diff
@@ -363,11 +363,18 @@ async def step_async(
                 self.cached_scheduler_outputs[
                     virtual_engine] = SchedulerOutputState()
 
+            current_step: Optional[int] = self._current_step(
+                seq_group_metadata_list)
+            # minus 1 because we updated the step count above
```

Comment:
Where did it get updated?

Reply:
inside `seq_group.finish_step()` . Updated the comment to better explain.